lib/assert.js

```diff
@@ -27,7 +27,7 @@ function formatWithLabel(label, value) {
 	return formatDescriptorWithLabel(label, concordance.describe(value, concordanceOptions));
 }
 
-const hasOwnProperty = (object, prop) => Object.prototype.hasOwnProperty.call(object, prop);
+export const hasOwnProperty = (object, prop) => Object.prototype.hasOwnProperty.call(object, prop);
```

Comment:
not sure we want to expose that here or place this in a more appropriate file (or duplicate worst case)

Reply:
It's available as `Object.hasOwn()` in Node.js 16 which will be the minimal required version soon enough. Repeating it is fine for now, I'm sure https://github.com/sindresorhus/eslint-plugin-unicorn will flag it soon enough 😉